nservicebus/testing/index.md

```diff
@@ -18,12 +18,12 @@ To install this package:
 ```
 Install-Package NServiceBus.Testing
 ```
-Once the package is installed you need ensure that you call `Test.Initialize()` (or any of its overloads) before executing any test method.
+Once the package is installed you need ensure that you call `Test.Initialize()` (or any of its overloads) before executing any test method. This is **not needed **for Version 6.
```

Comment:
Avoid language like 'you' which has a personal tone. 
Please see: https://github.com/particular/docs.particular.net#language-preferences

Also, it should be Versions 6 and above:
See: https://github.com/particular/docs.particular.net#version-language